.github/workflows/upload.yml

```diff
@@ -25,3 +25,9 @@ jobs:
           CHARMCRAFT_AUTH: "${{ secrets.CHARMCRAFT_AUTH }}"
         run: |
           charmcraft publish-lib charms.tls_certificates_interface.v2.tls_certificates
+      
+      - name: Publish libs
+        env:
+          CHARMCRAFT_AUTH: "${{ secrets.CHARMCRAFT_AUTH }}"
+        run: |
+          charmcraft publish-lib charms.tls_certificates_interface.v3.tls_certificates
```

Comment:
Will this actually work? will we have `charmcraft` installed on the system running this?
And should we consider using [the release-libraries](https://github.com/canonical/charming-actions/tree/main/release-libraries) action instead?

Reply:
I see the upload charm action will install the `charmcraft` snap based on previous runs.

Reply:
This was already being done before, he we simply add the v3 lib

Reply:
Alright, we can do it in a different PR then if needed :)